### PR TITLE
[EXAMPLE] Add custom growing animation on a Sequence Flow 

### PR DESCRIPTION
**Pull request by csouchet**
https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/example/add_custom_growing_edge/examples/index.html

### CODE REVIEW

File: examples/custom-animation/growing-sequence-flow/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Growing Sequence Flow
+
+Javascript example to demonstrate how to add custom growing animation on a Sequence Flow of the BPMN Diagram.
+- [__:fast_forward: live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/custom-animation/growing-sequence-flow/index.html)
+- to run locally, open the [index.html](index.html) directly in a Web Browser
+
+## ♻️ Usage
+
+After the main `BpmnVisualization` instantiation, get the HTML element corresponding to the sequence flow to add custom CSS class for the animation.
```
Comment (tbouffard) [inline examples/custom-animation/growing-sequence-flow/README.md:9]:
```suggestion
After the `BpmnVisualization` instantiation, get the HTML element corresponding to the sequence flow to add custom CSS class for the animation.
```

**Review by tbouffard — COMMENTED** [review]
LGTM, consistent with the other edge animation example 👍🏿

**Review by tbouffard — APPROVED** [review]

File: examples/custom-animation/growing-sequence-flow/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        stroke: Chartreuse;
+        animation-name: grow;
+        animation-duration: 2s;
+        animation-timing-function: ease-in-out;
+        animation-iteration-count: infinite;
+        animation-direction: normal;
+    }
+    @keyframes grow {
+        from {
+            stroke-dashoffset: 150;
```
Comment (aibcmars) [inline examples/custom-animation/growing-sequence-flow/README.md:25]: this value will need to be related to the length of the edge I guess
perhaps it is worth mentioning it?

Comment (csouchet) [reply examples/custom-animation/growing-sequence-flow/README.md:25]: I tried, but the rendering was dirty. So, with a value more longer is better.
We can imporve later 😉

Comment (tbouffard) [inline examples/custom-animation/growing-sequence-flow/README.md:25]:
```suggestion
            /* This value depends on the dimensions of the edge path */
            stroke-dashoffset: 150;
```

File: examples/custom-animation/growing-sequence-flow/index.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                stroke: Chartreuse;
+                animation-name: grow;
+                animation-duration: 2s;
+                animation-timing-function: ease-in-out;
+                animation-iteration-count: infinite;
+                animation-direction: normal;
+            }
+            @keyframes grow {
+                from {
+                    stroke-dashoffset: 150;
```
Comment (tbouffard) [inline examples/custom-animation/growing-sequence-flow/index.html:50]:
```suggestion
            /* This value depends on the dimensions of the edge path */
                    stroke-dashoffset: 150;
```

**Review by aibcmars — APPROVED** [review]
LGTM

**Pull request merged**